WeakAuras/WeakAuras.lua

```diff
@@ -1163,6 +1165,8 @@ loadedFrame:SetScript("OnEvent", function(self, event, addon)
     in_loading_screen = true;
   elseif(event == "LOADING_SCREEN_DISABLED") then
     in_loading_screen = false;
+  elseif(event == "UI_SCALE_CHANGED") then
+    Private:SetPixelMult()
```

Comment:
Well this needs to do a lot more, if we want to dynamically adjust to the uiscale. I think that's a bit too hard, and annoying.

I suggest we don't handle that. That is we react to the event, by printing that WAs will adjust on next ui reload. Otherwise you get very strange mix of some regions having updated to the new scale, and others not.